legacy/src/main/java/org/opensearch/sql/legacy/query/OpenSearchActionFactory.java

```diff
@@ -29,13 +28,7 @@
 import org.opensearch.client.Client;
 import org.opensearch.search.SearchHit;
 import org.opensearch.search.SearchHits;
-import org.opensearch.sql.common.setting.Settings;
-import org.opensearch.sql.legacy.domain.ColumnTypeProvider;
-import org.opensearch.sql.legacy.domain.Delete;
-import org.opensearch.sql.legacy.domain.IndexStatement;
-import org.opensearch.sql.legacy.domain.JoinSelect;
-import org.opensearch.sql.legacy.domain.QueryActionRequest;
-import org.opensearch.sql.legacy.domain.Select;
+import org.opensearch.sql.legacy.domain.*;
```

Comment:
same as previous comment

Reply:
Addressed in https://github.com/opensearch-project/sql/pull/3337/commits/84a2fffe7bbf78c738d70f0e430c4fdf49f567f8. Thanks!